lib/game_team.rb

```diff
@@ -2,13 +2,18 @@ class GameTeam
   attr_reader :game_id, :team_id, :tackles, :goals,
               :shots, :result, :head_coach
 
-   def initialize(raw_data)
+  def initialize(raw_data)
     @game_id = raw_data[:game_id]
     @team_id = raw_data[:team_id].strip
     @result = raw_data[:result]
     @head_coach = raw_data[:head_coach]
     @goals = raw_data[:goals].to_i
     @shots = raw_data[:shots].to_i
     @tackles = raw_data[:tackles].to_i
-   end
+  end
+
+  def is_game_pair?(game_team)
```

Comment:
Big fan of these methods that help out the managers

Reply:
Good job delegating to game_team